docs/common/ca.md

```diff
@@ -242,96 +206,3 @@ The index and change log files contain a summary of all the CA distributed and t
 | osg-ca-scripts | Stdout of osg-update-certs   | `/var/log/osg-ca-certs-status.system.out`       |
 | osg-ca-scripts | Stdout of osg-ca-manage      | `/var/log/osg-ca-manage.system.out`             |
 | osg-ca-scripts | Stdout of initial CA setup   | `/var/log/osg-setup-ca-certificates.system.out` |
-
-### Tests ###
-
-To test the host certificate of a server `openssl s_client` can be used. Here is an example with the gatekeeper:
```

Comment:
You might want to keep this bit or move it to a troubleshooting section somewhere.

Reply:
As it stands, it's fairly useless information: 

1. The command specifies `-cert` and `-key`, what happens if they don't have a host cert/key pair yet? What if the host cert/key pair are expired? Since this is a doc about CAs, these sorts of errors may lead people to think their CAs are broken when it's really the host cert/key.
1. What does good output look like?
1. What do they do if they get bad output?

I'll try and figure out something to make this useful but I'm still inclined to get rid of it entirely if I can't find something to work around the above issues.

Reply:
Fair enough. If you can do something for this, great; if you can't, that's fine too.

Reply:
The security team will be giving us a better command to include

Reply:
OK; we'll make that a separate PR.